src/components/SiteHeader/index.js

```diff
@@ -27,12 +27,12 @@ import useIsIncidentMap from 'hooks/useIsIncidentMap'
 import {
   IncidentMapHeader,
   IncidentMapHeaderWrapper,
-  IncidentMapMenuButton,
   Title,
   Wrapper,
 } from './incidentMapStyles'
 
 const MENU_BREAKPOINT = 1320
+const INCIDENT_MAP_MENU_BREAKPOINT = 540
```

Comment:
Is there a breakpoint defined somewhere like in ascDefaultTheme.breakpoints that's used more globally?